scripts/deps.py

```diff
@@ -1,24 +1,35 @@
 import sys
+import jinja2
+import yaml
 
-from conda_build import api
 
-section = {
-    'build': 'requirements/build',
-    'run':   'requirements/run',
-    'test':  'test/requires',
-}
+def load_setup_py_data():
+    import os
+    import setuptools
+    os.environ['CONDA_BUILD_STATE'] = 'RENDER'
+    data = {}
 
-names = set(sys.argv[1:])
+    def _setup(**kw): data.update(kw)
+    setuptools.setup = _setup
+    return data
 
-if not all(name in section for name in names):
-    print("Got unknown section name. Valid section names are: build, run, or test")
-    sys.exit(1)
+meta_src = jinja2.Template(open("conda.recipe/meta.yaml").read())
+meta_src = yaml.load(meta_src.render(load_setup_py_data=load_setup_py_data))
 
-metadata, _, _ = api.render('conda.recipe')
-
-deps = ""
+section = {
+    "build": meta_src["requirements"]["build"],
+    "run":   meta_src["requirements"]["run"],
+    "test":  meta_src["test"]["requires"],
+}
 
+spec = []
 for name in sys.argv[1:]:
-    deps += " ".join(s.replace(" ", "") for s in metadata.get_value(section[name])) + " "
+    spec += [s for s in section[name]]
```

Comment:
If the value of`section[name]` is a list, then I think `spec += section[name]` would do? Maybe it's not a list though, so I'm fine merging this as-is